compiler/hash-intrinsics/src/intrinsics.rs

```diff
@@ -749,10 +746,12 @@ impl DefinedIntrinsics {
                 "print_fn_directives",
                 FnTy::builder().params(params).return_ty(ret).build(),
                 |_, args| {
-                    if let Term::FnRef(fn_def_id) = args[1].value() {
-                        let directives =
-                            tir_stores().directives().get(fn_def_id.into()).unwrap_or_default();
-                        stream_less_writeln!("{:?}", directives.directives);
+                    if let Term::FnRef(_fn_def_id) = args[1].value() {
+                        todo!()
```

Comment:
left as Todo

Reply:
Yeah, this will be re-implemented once attributes are registered on respective nodes, and the terms have associated `AstNodeId`s

Reply:
I'll add a `@@ReAddDirectives` note so that this can be fixed when attributes are finally re-added.

Reply:
Okay, but maybe remove the todo!() so that it doesn't panic?